README.md

```diff
@@ -14,6 +14,18 @@ yarn install
 #  podman run  -it -p 8080:8080 apicurio/apicurio-registry-mem:latest-release
 REGISTRY_ENDPOINT="http://localhost:8080" yarn dev
 ```
+## Replicate the environment
+
+```sh
+# create a new environment
+python3.10 -m venv .env
+# activate it
+source .env/bin/activate
+# or this if on windows
```

Comment:
I would keep the windows instructions in the comment as ppl tend to copy-paste instructions directly in the terminal ...